setup.py

```diff
@@ -48,7 +48,8 @@ def get_long_description():
         "rouge>=1.0,<2",
         "sacrebleu>=1.4.3,<2",
         "tensorflow>=2.1,<2.2",
-        "tensorflow-addons>=0.8.1,<0.9"
+        "tensorflow-addons>=0.8.1,<0.9",
+        "pyter3>=0.3"
```

Comment:
As we don't know how stable this package is, let's use a strict requirement `pyter3==0.3`.

Reply:
Done.